index.js

```diff
@@ -30,7 +30,7 @@ function main(argv, callback) {
     ].join(''),
     argv);
 
-  var base = '.strong-pm';
+  var base = process.platform === 'win32' ? '\\\\.\\pipe' : '.strong-pm';
```

Comment:
`base` is what the app runs from, and is overridden by the `--base` CLI argument. I think you meant to change `control`, which is overridden by `--control`.